### ISSUE TITLE
Duplicate record

### ISSUE THREAD

**Issue by drupchen** [BuddhistDigitalResourceCenter/library-issues#210]
[G3957](https://www.tbrc.org/xmldoc?rid=G3957) and [G2301](https://www.tbrc.org/xmldoc?rid=G2301) seem to be duplicates.

The reason is explained in G3957: 
```
རྫོང་འདི་དང་སེར་ཤུལ་རྫོང་༼G2301༽གཉིས་དོན་གཅིག་ཡིན་ཏེ། རྒྱ་ཡིག་ནང་༼石渠县༽ཞེས་གཅིག་པ་འཁོད་ཡོད་ལ། འདིར་རྫོང་མིང་ཁག་ཁག་ཏུ་བཀོད་པ་ནི་དགོན་ཐོའི་ལོ་རྒྱུས་རྒྱུ་ཆ་W19997ནང་འཁོད་པ་གཞིར་བཟུང་བྱས་པ་ཡིན།
```

རྒྱ་ཡིག་ཙམ་དུ་མ་ཟད་པར་ས་མིང་གཉིས་ཡོད་པ་ལ་རྒྱུ་མཚན་ཡོད་པ་ནི། རྫ་ཆུ་ཁ་ནི་ཡོངས་གྲགས་ཀྱི་མིང་ཡིན་ལ། སེར་ཤུལ་དགོན་ཞེས་རྫོང་དེའི་ཁོངས་གཏོགས་སུ་ཡོད་པ་ལ་དཔོག་ན་སེར་ཤུལ་ཞེས་པའི་ས་མིང་ཡང་རྒྱུ་མཚན་མེད་པ་མིན་ནམ་སྙམ། 

དེ་ཙམ་དུ་མ་ཟད་པར་ཝི་ཁེ་རིག་མཛོད་ལ་ཆ་བཞག་ན་ས་མིང་གཉིས་པོ་ཤོག་ངོས་གཅིག་གི་ནང་དུ་བཀོད་ཡོད་པ་ནི་[འདིར་གཟིགས་འཚལ](https://en.wikipedia.org/wiki/S%C3%AArx%C3%BC_County)།

**Comment by karmagongde** [comment]
I have a discussion with Librarians about this topic and many times geographical name changes time to time and in a different era, the same place has a different names. I also discussed with ED Jann about this issue and he suggests we should wait until the BUDA platform is done.

**Comment by xristy** [comment]
Yes. It was obvious that the duplication was intentional and we need to create an appropriate model for the changes in place names and regions, usage and so on over time. 

It would be helpful if there was additional notes for these two place records regarding why the intentional duplication - what era or other time frame does each belong to or what other reason there is for the duplication.

Also it would be useful if the note in G3957 is also presented in English as well as the current Tibetan.

Finally, it would be most helpful if you could develop a list of additional place duplicates or multiplicities that have been entered with the reasons, dates and such. This is being requested so that there is as much information as possible to help inform the development of an appropriate model for place timelines

**Comment by drupchen** [comment]
I would say that it would be interesting to all users if all the notes could be presented in English, as the information they contain is very helpful, yet unavailable to users not fluent in written Tibetan.
